Sources/Tools/plNetLog/ChunkBuffer.h

```diff
@@ -50,6 +50,12 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include <QString>
 #include <QMutex>
 
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+#define Q_MUTEX_LOCKER_TYPE QMutexLocker<QMutex>
+#else
+#define Q_MUTEX_LOCKER_TYPE QMutexLocker
+#endif
```

Comment:
This could be done with `typedef`/`using`, right?

```suggestion
#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
using = Q_MUTEX_LOCKER_TYPE QMutexLocker<QMutex>;
#else
using Q_MUTEX_LOCKER_TYPE = QMutexLocker;
#endif
```

Reply:
Probably... let me give it a shot